storage-xray-udp/src/main/java/zipkin2/storage/xray_udp/UDPMessageEncoder.java

```diff
@@ -55,8 +57,15 @@
         || span.kind() != Span.Kind.SERVER && span.kind() != Span.Kind.CONSUMER) {
       writer.name("type").value("subsegment");
       if (span.kind() != null) writer.name("namespace").value("remote");
+      // some remote service's name can be null, using null names causes invisible subsegment
+      // using "unknown" subsegment name will help to detect missing names
+      writer.name("name").value(span.remoteServiceName() == null ? "unknown" : span.remoteServiceName());
+    }else{
+      writer.name("name").value(span.localServiceName());
```

Comment:
hey, I am now getting all my traces labelled as "unkown", previously they used to obey the `localServiceName()`. How is remoteServiceName supposed to be set? I usually set the localServiceName through `Tracing.newBuilder().localServiceName`, but there is no similar method for remote service name.